video/out/wayland_common.c

```diff
@@ -423,6 +423,8 @@ static void keyboard_handle_leave(void *data, struct wl_keyboard *wl_keyboard,
 {
     struct vo_wayland_state *wl = data;
     wl->has_keyboard_input = false;
+    wl->mpkey = 0;
```

Comment:
This doesn't appear to actually be resetting keys or modifiers on sway at least. I checked with logs, mpv does receive a keyboard leave handle

Reply:
Ah yes you're right. I also missed the wl->keyboard_code